add-ons/debezium/debezium.addon

```diff
@@ -0,0 +1,17 @@
+# Name: debezium
+# Description: Deploys a Kafka Cluster and a Connect cluster with Debezium
+# Required-Vars: DEBEZIUM_VERSION,DEBEZIUM_PLUGIN,PROJECT
+
+# Install templates
+oc create -f https://raw.githubusercontent.com/EnMasseProject/barnabas/master/kafka-statefulsets/resources/openshift-template.yaml -n #{PROJECT}
+oc create -f https://raw.githubusercontent.com/EnMasseProject/barnabas/master/kafka-connect/s2i/resources/openshift-template.yaml -n #{PROJECT}
```

Comment:
Are you sure you want to point to the master and not the released version? Also IMO it is a good practice to put the local copies of the templates in the add-on itself instead of pointing to the web for getting it to avoid cases when the files do not work because of a bad commit in the repository.

Reply:
At the time of creating the add-on there was no tag yet. There is one already so we are goint to use that one.